src/Oci8/Oci8Connection.php

```diff
@@ -196,7 +197,6 @@ public function getDoctrineConnection()
                 $data, $this->getDoctrineDriver()
             );
         }
-        
```

Comment:
I think the new line here is needed. No newline on return doc block but newline is needed for function returns.